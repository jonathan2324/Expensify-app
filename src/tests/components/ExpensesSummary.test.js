import React from 'react'
import { shallow } from 'enzyme'
import expenses from '../fixtures/expenses'
import {ExpensesSummary} from '../../components/ExpensesSummary'

test("Should render ExpensesSummary page 1", () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235}/>)
    expect(wrapper).toMatchSnapshot()
})

test("Should render ExpensesSummary page mult", () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={23500}/>)
    expect(wrapper).toMatchSnapshot()
})