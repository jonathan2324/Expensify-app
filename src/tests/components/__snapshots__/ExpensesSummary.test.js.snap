// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpensesSummary page 1 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`Should render ExpensesSummary page mult 1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $235.00
  </h1>
</div>
`;
